code/datums/station_traits/neutral_traits.dm

```diff
@@ -127,3 +127,95 @@
 
 	var/new_colored_assistant_type = pick(subtypesof(/datum/colored_assistant) - get_configured_colored_assistant_type())
 	GLOB.colored_assistant = new new_colored_assistant_type
+
+/datum/station_trait/new_uniform_standards
+	name = "New Uniform Standards: Base Trait (Shouldn't Run)"
+	trait_type = STATION_TRAIT_NEUTRAL
+	weight = 0
+	show_in_report = TRUE
+	report_message = "We've issued some new uniforms for the crew."
+	trait_to_give = STATION_TRAIT_NEW_UNIFORM_STANDARDS
+	var/list/uniforms = list("command" = /datum/outfit/job/command_trek,
+							"engsec" = /datum/outfit/job/engsec_trek,
+							"medsci" = /datum/outfit/job/medsci_trek,
+							"srvcar" = /datum/outfit/job/srvcar_trek)
+
+/datum/station_trait/new_uniform_standards/tos
+	name = "New Uniform Standards: TOS"
+	weight = 10
```

Comment:
altogether this trait has 30 weight, 
all negative events have 2-5 weight
all positive events have 1-5 weight with two 10 weight ones
all neutral traits have 5 weight with a 10 and 15 one
this is way too high for something that fucks over a lot of people

Reply:
adjusting it down to weight of 4